bindings/python/test/test_arrow.py

```diff
@@ -384,3 +387,84 @@ def test_find_decimal128(self):
         )
         table = find_arrow_all(coll, {}, schema=schema)
         self.assertEqual(table, expected)
+
+
+class TestNulls(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls, find_fn=find_arrow_all):
+        if not client_context.connected:
+            raise unittest.SkipTest("cannot connect to MongoDB")
+        cls.cmd_listener = AllowListEventListener("find", "aggregate")
+        cls.getmore_listener = AllowListEventListener("getMore")
+        cls.client = client_context.get_client(
+            event_listeners=[cls.getmore_listener, cls.cmd_listener]
+        )
+
+        cls.oids = [ObjectId() for _ in range(4)]
+
+        # Default integral types
+        cls.int_schema = Schema({"_id": ObjectIdType(), "int64": int64()})
```

Comment:
These arrays and schemas are only used in their respective test, could they be moved to local variables?